src/cli/command_list.py

```diff
@@ -192,7 +209,7 @@ def experiments_delete() -> None:
 @experiments_app.command("run")
 def experiments_run(
     block: bool = typer.Option(
-        True, "--block", help="Wait for the result to be returned."
+        False, "--block", help="Wait for the result to be returned."
```

Comment:
Parameter only for remote I suppose? Or are we running asynchronous also local?
The requirements says default is True, why change it to False?